mail_notification_custom_subject/i18n/mail_notification_custom_subject.pot

```diff
@@ -128,7 +130,29 @@ msgid "Templates"
 msgstr ""
 
 #. module: mail_notification_custom_subject
-#: model:ir.model.fields,help:mail_notification_custom_subject.field_mail_message_custom_subject__position
-msgid "Whether to replace, append at beggining or append at end to other templates that apply to a given context"
+#: code:addons/mail_notification_custom_subject/tests/test_mail_notification_custom_subject.py:0
+#: code:addons/mail_notification_custom_subject/tests/test_mail_notification_custom_subject.py:0
+#: code:addons/mail_notification_custom_subject/tests/test_mail_notification_custom_subject.py:0
+#: code:addons/mail_notification_custom_subject/tests/test_mail_notification_custom_subject.py:0
+#: code:addons/mail_notification_custom_subject/tests/test_mail_notification_custom_subject.py:0
+#: code:addons/mail_notification_custom_subject/tests/test_mail_notification_custom_subject.py:0
+#: code:addons/mail_notification_custom_subject/tests/test_mail_notification_custom_subject.py:0
+#: code:addons/mail_notification_custom_subject/tests/test_mail_notification_custom_subject.py:0
+#: code:addons/mail_notification_custom_subject/tests/test_mail_notification_custom_subject.py:0
+#: code:addons/mail_notification_custom_subject/tests/test_mail_notification_custom_subject.py:0
+#, python-format
+msgid "Test"
+msgstr ""
+
+#. module: mail_notification_custom_subject
+#: code:addons/mail_notification_custom_subject/tests/test_mail_notification_custom_subject.py:0
+#, python-format
+msgid "Test partner 1"
 msgstr ""
```

Comment:
Why did this strings get added to the translation files? They are only used in tests...

Reply:
These translations were added because precommit force me to set `_( )` to each term. When I extract the pot file from Odoo, the terms were added

Reply:
Mmm ok :thinking: Although it is weird that pre-commit makes you do that... I guess it isn't a huge problem, anyway...

Reply:
I will avoid the error by pylint: disable